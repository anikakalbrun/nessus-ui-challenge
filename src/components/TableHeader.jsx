import React from 'react';

export default function TableHeader() {
    return (
        <h1 className="App-intro m-t-50 m-b-20 center">Server Configurations are listed below.</h1>
    );
}