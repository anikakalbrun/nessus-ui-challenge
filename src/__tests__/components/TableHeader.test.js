import React from 'react';
import TableHeader from '../../components/TableHeader';

let wrapper;

describe('TableHeader', () => {

    it('renders correctly', () => {
        wrapper = renderer.create(<TableHeader/>).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});
