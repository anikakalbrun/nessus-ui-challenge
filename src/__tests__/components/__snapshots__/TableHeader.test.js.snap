// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader renders correctly 1`] = `
<h1
  className="App-intro m-t-50 m-b-20 center"
>
  Server Configurations are listed below.
</h1>
`;
